content/integrations/docker.md

```diff
@@ -12,56 +12,99 @@ git_integration_title: docker
-
-**Note:** docker_daemon replaces the older docker integration going forward.
-
-# Installation
-
 ## Host Installation
 
 1. Ensure Docker is running on the host.
 2. Install the agent as described in [the agent installation instructions](https://app.datadoghq.com/account/settings#agent) for your host OS.
 3. Enable [the Docker integration tile in the application](https://app.datadoghq.com/account/settings#integrations/docker).
-4. Add the agent user to the docker group: ```usermod -a -G docker dd-agent```
-5. Create  **docker_daemon.yaml** by copying the example file in the agent conf.d directory. If you have a standard install of Docker on your host, there shouldn't be anything you need to change to get the integration to work.
-6. To enable other integrations, use ```docker ps``` to identify the ports used by the corresponding applications.
+4. Add the agent user to the docker group: `usermod -a -G docker dd-agent`
+5. Create a `docker_daemon.yaml` file by copying [the example file in the agent conf.d directory](https://github.com/DataDog/dd-agent/blob/master/conf.d/docker_daemon.yaml.example). If you have a standard install of Docker on your host, there shouldn't be anything you need to change to get the integration to work.
+6. To enable other integrations, use `docker ps` to identify the ports used by the corresponding applications.
     ![](/static/images/integrations-docker-dockerps.png)
 
+<%= insert_example_links(conf: "docker_daemon", check: "docker_daemon")%>
+
+**Note:** docker_daemon has replaced the older docker integration.
+
 ## Single Container Installation
 
 1. Ensure Docker is running on the host.
-2. Install the Docker container as described in [the docker container installation instructions](https://app.datadoghq.com/account/settings#agent/docker).
+2. As per [the docker container installation instructions](https://app.datadoghq.com/account/settings#agent/docker), run:
+
+        docker run -d --name dd-agent \
+          -v /var/run/docker.sock:/var/run/docker.sock:ro \
+          -v /proc/:/host/proc/:ro \
+          -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+          -e API_KEY={YOUR API KEY} \
+          datadog/docker-dd-agent:latest
 
 ### Environment variables
 
-A few parameters can be changed with environment variables.
+Note that in the command above, you are able to pass your API key to the Datadog Agent using Docker's `-e` or environment variable flag. Some other variables you can pass include:
+
+* **TAGS** sets host tags. You can pass both simple tags and key-value tags. For example, `-e TAGS="simple-tag,tag-key:tag-value"`.
+* **LOG_LEVEL** sets logging verbosity (CRITICAL, ERROR, WARNING, INFO, DEBUG). For example, `-e LOG_LEVEL=DEBUG` will set logging to debug mode.
+* **PROXY_HOST**, **PROXY_PORT**, **PROXY_USER**, and **PROXY_PASSWORD** set the proxy configuration.
+* **DD_URL** sets the Datadog intake server where the agent will send metrics (used when using an agent as a proxy).
```

Comment:
Thanks. Yeah, I think we should add the SD stuff in passing and point to the SD docs. I'll also add the other env vars.